lib/paramedic.js

```diff
@@ -109,7 +109,7 @@ class ParamedicRunner {
                         });
                 }
 
-                return this.paramedicSauceLabs.displaySauceDetails(this.sauceBuildName);
+                return this.paramedicSauceLabs.displaySauceDetails.apply(this.paramedicSauceLabs, [this.sauceBuildName]); // eslint-disable-line
```

Comment:
And maybe this should be merged together with the other scope change commit. As well, all the other changes bellow where you `apply` the `this.paramedicSauceLabs)` scope.

Reply:
Yes, I will do this as well.